static/js/stat_var_hierarchy/stat_var_section_input.tsx

```diff
@@ -97,12 +106,53 @@ export class StatVarSectionInput extends React.Component<
         <label
           className={this.state.checked ? "selected-node-title" : ""}
           htmlFor={sectionId}
+          onMouseMove={
+            !this.props.statVar.hasData ? this.mouseMoveAction : null
+          }
+          onMouseOut={() => hideTooltip()}
         >
           {this.props.statVar.displayName}
         </label>
       </form>
     );
   }
+
+  private mouseMoveAction = (e) => {
+    let html = "loading stat var summary...";
+    if (!_.isEmpty(this.props.summary)) {
+      let availablePlaceTypes = Object.keys(
+        this.props.summary.placeTypeSummary
+      );
+      if (this.context.statVarHierarchyType === StatVarHierarchyType.MAP) {
+        availablePlaceTypes = availablePlaceTypes.filter(
+          (placeType) =>
+            placeType in USA_CHILD_PLACE_TYPES && placeType !== "Country"
+        );
+      }
+      html =
+        availablePlaceTypes.length > 0
+          ? "This stat var has no data for any of the chosen places." +
+            "You can try these types of places instead. <ul>"
+          : "Sorry, this stat var is not supported by this tool.";
+      for (const placeType of availablePlaceTypes) {
+        const placeSummary = this.props.summary.placeTypeSummary[placeType];
+        const placeList = placeSummary.topPlaces.map((place) => place.name);
+        html +=
```

Comment:
do you think it works to make the place names hyper link? Then user can directly go to the timeline page of that place.

Reply:
I think we've discussed how links don't really belong in tooltips..

Reply:
I think we've discussed that tooltips aren't really supposed to be used for the user to take actions? It should be more of a simple informational kind of thing ..

Reply:
I can see that, but from usability point of view, navigating to the hint places take some more effort (search place, then find the stat vars). We can gather thoughts from the team too, can update in later PR if necessary.

Reply:
Yeah, I see what you mean, but in that case, maybe tooltip isn't the best way to show this information. Also trying to give users examples to navigate to will be a little more complicated for scatter and map tools where we need to know the parent place.

But yes, let's gather thoughts from the team!